src/Desktop/Styles/StyleId.cs

```diff
@@ -20,21 +20,21 @@ public class StyleId : TypeBase
         /// https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-style-identifiers
         /// </summary>
 #pragma warning disable CA1707 // Identifiers should not contain underscores
-        public const int StyleId_Custom       = 70000;
```

Comment:
I kind of like the alignment, but I'm willing to give it up to get the rest of the code consistently formatted

Reply:
Totally with you on this but I'm not sure how easy it would be to work against the formatter in the long term.